latest/examples/multiscales_strict/multiscales_example.json

```diff
@@ -3,44 +3,50 @@
             "version": "0.5-dev",
             "name": "example",
-            "axes": [
-                {"name": "t", "type": "time", "unit": "millisecond"},
-                {"name": "c", "type": "channel"},
-                {"name": "z", "type": "space", "unit": "micrometer"},
-                {"name": "y", "type": "space", "unit": "micrometer"},
-                {"name": "x", "type": "space", "unit": "micrometer"}
+            "coordinateSystems" : [
+                {
+                    "name" : "example",
+                    "axes": [
+                        {"name": "t", "type": "time", "unit": "millisecond"},
+                        {"name": "c", "type": "channel"},
+                        {"name": "z", "type": "space", "unit": "micrometer"},
+                        {"name": "y", "type": "space", "unit": "micrometer"},
+                        {"name": "x", "type": "space", "unit": "micrometer"}
+                    ]
+                }
             ],
             "datasets": [
                 {
                     "path": "0",
                     "coordinateTransformations": [{
-                        // the voxel size for the first scale level (0.5 micrometer)
+                        // the voxel size for the first scale level (0.5 micrometer), time unit (0.1 milliseconds)
                         "type": "scale",
-                        "scale": [1.0, 1.0, 0.5, 0.5, 0.5]
+                        "scale": [0.1, 1.0, 0.5, 0.5, 0.5],
+                        "input" : "/0",
+                        "output" : "example"
                     }]
                 },
                 {
                     "path": "1",
                     "coordinateTransformations": [{
-                        // the voxel size for the second scale level (downscaled by a factor of 2 -> 1 micrometer)
+                        // the voxel size for the second scale level (downscaled by a factor of 2 -> 1 micrometer), time unit (0.1 milliseconds)
                         "type": "scale",
-                        "scale": [1.0, 1.0, 1.0, 1.0, 1.0]
+                        "scale": [0.1, 1.0, 1.0, 1.0, 1.0],
+                        "input" : "/1`",
+                        "output" : "example"
                     }]
                 },
                 {
                     "path": "2",
                     "coordinateTransformations": [{
-                        // the voxel size for the third scale level (downscaled by a factor of 4 -> 2 micrometer)
+                        // the voxel size for the third scale level (downscaled by a factor of 4 -> 2 micrometer), time unit (0.1 milliseconds)
                         "type": "scale",
-                        "scale": [1.0, 1.0, 2.0, 2.0, 2.0]
+                        "scale": [0.1, 1.0, 2.0, 2.0, 2.0],
+                        "input" : "/2",
+                        "output" : "example"
                     }]
                 }
             ],
-            "coordinateTransformations": [{
```

Comment:
Is the top-level multiscales coordinateTransformations no-longer valid? If it is, then this is a useful example. If not then the spec wording needs to be changed.